### ISSUE TITLE
Help menu "learn more" option crashes the browsers

### ISSUE THREAD

**Issue by DoodlesEpic** [probablykasper/kadium#4]
When trying to open the "learn more" option on the help thingy my browser crashes instead of opening the window for help page:

![image](https://user-images.githubusercontent.com/37254797/176284530-5f53db8d-d99c-47aa-90e5-a906f0568718.png)
![image](https://user-images.githubusercontent.com/37254797/176284683-2325f804-76b4-4ef6-9e08-ba52192e0db8.png)

For some reason I couldn't copy the error report by ABRT so I just screenshot, I hope this helps tracing the bug. Oh and I should note I am using the AppImage and not the deb package as it's Fedora.

**Comment by DoodlesEpic** [comment]
Oh well clicking on channels is also giving the same problem now, possibly all of the anchor tags are throwing errors then. It's probably a problem with Tauri itself, maybe using [tauri-plugin-deep-link](https://github.com/FabianLars/tauri-plugin-deep-link) could work.

**Comment by probablykasper** [comment]
Just updated the app to 1.4.0, does that fix it? If not, my guess is that it's an issue with Tauri's AppImage building :/

**Comment by DoodlesEpic** [comment]
Nope, problem persists on 1.4.0. Probably a problem in Tauri indeed

**Comment by probablykasper** [comment]
Does it happen if you switch your default browser to Firefox?

Would you be able to test if it happens with the AppImage on a Debian-based distro?

**Comment by DoodlesEpic** [comment]
So the result is very strange when using Firefox, the application "crashes" but the tab opens and stays open which is very funny. I don't have Debian-based distro setup currently, will see if I manage to get a virtual machine going.

![image](https://user-images.githubusercontent.com/37254797/176565596-d6fef4a9-528d-4f9f-9a5b-bf058889b9a7.png)
![image](https://user-images.githubusercontent.com/37254797/176565615-e174124b-a0ad-402c-8aa4-da4794bc0010.png)

**Comment by DoodlesEpic** [comment]
On Debian Testing both Brave Browser and Firefox work perfectly using the AppImage. I'm curious as to why this might be then. Differences between the system include the fact that my Fedora install is using Wayland and Pipewire on a NVIDIA graphics card which forces the browsers to start with a few parameters, namely --ozone-platform=wayland on chromium. I will test this on a Fedora virtual machine next to see what happens, but this is really a problem with Tauri it seems like, not sure Kadium can do much to mitigate.

![image](https://user-images.githubusercontent.com/37254797/176581141-3a13be8f-a3fb-4d30-8676-288b3f93c188.png)

**Comment by DoodlesEpic** [comment]
Fedora Workstation 36 live install using GNOME Boxed (QEMU) works out of the box without ABRT throwing any errors, so this is very likely related to NVIDIA-something on the system. Not sure where how we should report this to Tauri though? I'll create a temporary issue on the Tauri GitHub but I won't close this for now as I'll be referencing this there. Thank you very much @probablykasper, you are amazing with CSS btw, love your websites.

![image](https://user-images.githubusercontent.com/37254797/176583699-fb4e6093-fde0-405e-81b7-66bc4d73337c.png)

**Comment by DoodlesEpic** [comment]
Create the bug report on the Tauri repo
https://github.com/tauri-apps/tauri/issues/4526

**Comment by probablykasper** [comment]
Great, I think it'll be best to continue discussion over there because it's not something that should really be fixed within Kadium itself

> Thank you very much @probablykasper, you are amazing with CSS btw, love your websites.

haha thanks! :DD
